### PR TITLE
Auditor Qualification Enums

### PR DESCRIPTION
**Pull request by nllong**
See proposals/2019/Qualifications List V2.md.

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -15444,6 +15445,11 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
       <xs:enumeration value="ASHRAE Building Energy Assessment Professional (BEAP)"/>
       <xs:enumeration value="Department of Buildings (DOB) Approved Agent"/>
       <xs:enumeration value="High-Performance Building Design Professional (HBDP)"/>
+      <xs:enumeration value="GreenPoint Rater Existing Home Multifamily Rater"/>
+      <xs:enumeration value="HERS Whole House (HERS II) Rater"/>
+      <xs:enumeration value="International Union of Operating Engineers Certified Energy Specialist"/>
+      <xs:enumeration value="Northwest Energy Education Institute Energy Management Certification"/>
+      <xs:enumeration value="PhD in Mechanical Engineering"/>
```
Comment (nllong) [inline BuildingSync.xsd]: @markborkum, @supriyagoel  -- what is the audit program that is accepting this as a qualification? This seems very generic and doesn't necessarily qualify a person to conduct an audit.

Comment (edward-ellis) [reply BuildingSync.xsd]: The cities of San Francisco and Berkeley accept this a the qualification for Energy Auditor.

Comment (markborkum) [reply BuildingSync.xsd]: @nllong The audit programs for the cities of San Francisco and Berkeley accept "PhD in Mechanical Engineering" qualifications for the role of "Energy Auditor".

Comment (nllong) [reply BuildingSync.xsd]: Okay, thanks gentlemen. I'll leave it in there then.

**Pull request merged**